NuRadioReco/modules/LOFAR/stationPulseFinder.py

```diff
@@ -261,7 +262,7 @@ def run(self, event, detector):
             The detector related to the event.
         """
         zenith = event.get_hybrid_information().get_hybrid_shower("LORA").get_parameter(showerParameters.zenith)
-        azimuth = event.get_hybrid_information().get_hybrid_shower("LORA").get_parameter(showerParameters.azimuth)
+        azimuth = np.mod( event.get_hybrid_information().get_hybrid_shower("LORA").get_parameter(showerParameters.azimuth), 2* np.pi )
```

Comment:
Is this really necessary or just a remain of trying different solutions?

Reply:
I think it is just a remainder. I will remove it.